pkg/datapath/route/route_darwin.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +build darwin
+
+package route
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/sirupsen/logrus"
+	"github.com/vishvananda/netlink"
+)
+
+type Route struct {
```

Comment:
Is there a way to share the core route structure (and perhaps the `getLogger()` and other platform-independent code) in a common route.go then only have the platform-specific bits in `route_xxx.go`?

Reply:
Will do! Thanks for the review.

Reply:
Done.